### DISCUSSION TITLE
Where to find the printer G-code file?

### DISCUSSION THREAD

**Discussion by olaab** [OrcaSlicer/OrcaSlicer#5144] [General]
I'm looking for the printer, ex. Prusa G-code file. I want to change the piezo buzzer frequency. Where can I find the file?

**Comment by LJKraus** [comment]
Probably want to edit the Printer settings, Machine Start  G-code:
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/60d7a32d-99ca-47e4-939e-ad9283460bd3)

**Reply by olaab** [reply]
Nope you don't find it here. 
There must be a file some where who have a lot of code G-code, where else should the buzzer code come from and other code. 

Do you have the buzzer code there in machin code window?

**Comment by LJKraus** [comment]
You might find something in the Configuration folder:
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/5bc7ad72-d9f6-44b9-82b7-d0c7ca951fb4)

There are no files with a .gcode extension. Gcode commands are in various .json files that the slicer adds to generated gcode where appropriate..  

The only references I find to changing Prusa buzzer frequency use [M300 commands](https://help.prusa3d.com/article/prusa-firmware-specific-g-code-commands_112173#m-commands:~:text=M300%20%2D%20Play%20tone,duration%20in%20milliseconds) added to Machine Start and End code in PrusaSlicer, using [editing methods much like OrcaSlicer](https://forum.prusa3d.com/forum/postid/99205/).

Do not edit the default files - they will be replaced with every software update.  Changes should be saved in a user preset, which will not be overwritten during an update.

Sorry if this does not help, I don't have a Prusa, and my printer does not buzz.

**Reply by olaab** [reply]
OK, thanks! 
I don't know but what ever I do I get this buzzer. When I try to put code in the end G-code field nothing happens. It still following som code in the background. 
Ex. I want to rise the nozzle 10 mm. Set up the code who works nice in simplify3d but here nothing happens, even if I can see it in the code window! 
Very irritating! 

Win 10
OC 2.0
Marlin,  Greeetech base home built.
